src/main/scala/io/buoyant/linkerd/RateLimiterConfig.scala

```diff
@@ -0,0 +1,23 @@
+package io.buoyant.linkerd
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.finagle.util.DefaultTimer
+import com.twitter.finagle.{Filter, Stack}
+import io.buoyant.linkerd.protocol.HttpRequestAuthorizerConfig
+
+class RateLimiterConfig(limit: Int, window: Int) extends HttpRequestAuthorizerConfig{
+  @JsonIgnore
+  override def role: Stack.Role = Stack.Role("HttpRateLimiter")
+
+  @JsonIgnore
+  override def description: String = "HTTP Rate Limiter"
+
+  @JsonIgnore
+  override def parameters: Seq[Stack.Param[_]] = Seq()
+
+  @JsonIgnore
+  override def mk(params: Stack.Params): Filter[Request, Response, Request, Response] = {
+    new RateLimiter(limit, DefaultTimer, window)
```

Comment:
What happens if `limit <= 0`. What happens if `window is <= 0` ?

Reply:
We now require both `limit` and `intervalSeconds` to be greater than 0. This is checked in `RateLimiterConfig` because that is where users will interact with creating a new `RateLimiter`.